sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientTest.java

```diff
@@ -30,9 +30,7 @@
 import static com.azure.ai.formrecognizer.TestUtils.BLANK_PDF;
```

Comment:
I think for FR it was a suggestion to not assert any of the service side error messages since they can update that any time. It would be best to remove all such error messages asserts.

Reply:
done